Add-ons/UmbracoForms/Editor/Creating-a-Form/Conditional-Logic/index.md

```diff
@@ -1,58 +1,55 @@
 # Setting up conditional logic on fields
 
-Only showing a field if the value of another field is ... That's what you can do with conditional logic.
+Some times you might have a field in your form, that you only want to show if the user has entered a specific value in another field: 
 
-## Example form
-Take a look at the following as an example
+*Only show this field if the value of that field is x*
 
-![Example form](ExampleForm.png)
+You can achieve this by setting up **conditional logic**.
 
-It make sense to only show the email/phone field when that corresponds with the option chosen in the 'how should we contact you?' field.
+## Example
 
-## Enabling conditional logic
+Take a look at the following as an example
 
-Enabling conditions for a field can be done on the additional settings of a field.
+![Example form](images/ExampleForm.png)
 
+In this case it would make sense to **only show the email/phone field when that corresponds with the option chosen in the 'How should we contact you?' field**.
 
-![Enable conditions](EnableConditions.png)
+The achieve that, we need to enabling conditions for the **Email** and **Phone** fields. This can be done in the additional settings.
```

Comment:
```suggestion
To achieve that, we need to enable conditions for the **Email** and **Phone** fields. This can be done in the additional settings.
```